### PR TITLE
add rough overview of resource limits

### PR DESCRIPTION
**Pull request by linki**

### CODE REVIEW

**Comment by szuecs** [comment]
@linki Can you please make sure that you right with a capital letter if you start a sentence?

**Comment by linki** [comment]
@szuecs fixed

File: docs/user-guide/resource-limits.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+=========================
+Container resource limits
+=========================
+
+*Note: preliminary summary from skimming docs and educational guessing.*
+*No evaluation done. It could contain errors.*
```
Comment (mikkeloscar) [inline docs/user-guide/resource-limits.rst]: Should this be a [sphinx note](http://www.sphinx-doc.org/en/1.4.8/markup/para.html#directive-note)? :)

File: docs/user-guide/resource-limits.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Maps to the docker flag ``--cpu-shares``, which defines a relative weight of that container
+for cpu time. The relative share is executed per core, which can lead to unexpected outcomes
+but probably nothing to worry about in our use cases. A container will never be killed
+because of this metric.
+
+Resource limits
+---------------
+
+Limits define the upper bound of resources a container can use. Limits must always be greater
+equal than requests. The behaviour differs between cpu and memory.
```
Comment (mikkeloscar) [inline docs/user-guide/resource-limits.rst]: > Limits must always be **greater equal than** requests

greater or equal to

**Comment by linki** [comment]
@mikkeloscar @szuecs fixed

**Comment by hjacobs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
@linki you need to also reference this section in the index.rst: https://raw.githubusercontent.com/zalando-incubator/kubernetes-on-aws/master/docs/user-guide/index.rst

**Comment by linki** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**